assets/stylesheets/pages/_reactivex.scss

```
@@ -0,0 +1,51 @@
+._reactivex {
+  @extend %simple;
+
+  img {
+    max-width: 50%;
+  }
+
+  figure {
+    margin: 0;
+  }
+
+  dfn {
+    cursor: text;
+    font-style: italic;
+    text-decoration: none;
+    border-bottom: none;
+  }
+
+  #tree {
+    dt, dd {
+      font-weight: normal;
+    }
+
+    dt {
+      float: left;
+      clear: left;
+
+      margin-top: 0;
+
+      &:before {
+        content: "…";
```

```suggestion
        content: "\2026";
```

Seems Rails doesn’t like non-ASCII content in assets.